src/features/UserList/UserList.tsx

```diff
@@ -32,6 +56,14 @@ const UserList: React.FC<UserListProps> = ({ title }) => {
   const classes = useStyles();
```

Comment:
It seems like the component starts to deal with many responsibilities at once, let's try to decompose it?